patches/minecraft/net/minecraft/crash/CrashReport.java.patch

```diff
@@ -8,24 +8,24 @@
     }
  
     public String func_71501_a() {
-@@ -87,14 +88,8 @@
+@@ -87,14 +88,10 @@
        if (this.field_85060_g != null && this.field_85060_g.length > 0) {
           p_71506_1_.append("-- Head --\n");
           p_71506_1_.append("Thread: ").append(Thread.currentThread().getName()).append("\n");
 -         p_71506_1_.append("Stacktrace:\n");
--
+ 
```

Comment:
I think that you get that whitespace REALLY isn't the issue in patches

Reply:
I can't tell if you want me to undo the whitespace changes or if you're just telling me this for future reference...